theme-dev/README.md

```diff
@@ -111,29 +113,29 @@ This is for security and privacy reasons.
 
 
 * github.com
-* raw.githubusercontent.com
+* raw.githubusercontent.com 
 * gitlab.com
-* cdn.discordapp.com
-* media.discordapp.net
+* cdn.discordapp.com **(expires)**
```

Comment:
Linking to ![this](https://discord.com/channels/811255666990907402/857624431148138518/1210912754471411732) would be a good idea, maybe removing it would be a good idea in general and just leaving a notice above?

Reply:
got it